ci-scripts/run-integration-tests.sh

```diff
@@ -3,8 +3,8 @@
 set -e
 
 # Build and start chain Devnet
-git clone https://github.com/calvinlauco/chain || echo "`chain` folder already exists"
-cd chain; git checkout feature/cro-629-travis-integration-tests; git pull; cd ..;
+git clone https://github.com/crypto-com/chain || echo "`chain` folder already exists"
+cd chain; git checkout master; git pull; cd ..;
```

Comment:
The old way of running integration tests is deprecated in `chain` repository. While migration to the new drone.io would take some time (and is in backlog), consider pointing the integration tests to:

https://github.com/crypto-com/chain/tree/438b62c07a3d91760d1bebabc583af8c9242ca16/integration-tests

at the moment.

Reply:
Sure thing~